APIBacklog/documentation/SupportingDocuments/API proposals/APIFamilyproposal_Identity&Consent.md

```diff
@@ -3,7 +3,7 @@
 | API family name | Identity and Consent Management |
 | API family owner| Telefonica  |
 | API family summary | **High level description**<br>Operator (NaaS) platform implementing CAMARA APIs should be built with a privacy-by-default approach to fully comply with data protection regulations like the [GDPR regulation](https://gdpr-info.eu) in Europe, which emphasises on user's privacy. These regulations note that some CAMARA APIs may require user consent to be accessed. This forces the operators to provide means and appropriate solutions to capture, store and manage this consent through its lifecycle. Otherwise, the scoped CAMARA APIs cannot be rolled out in production networks. <br>Building such a solution also means bringing in scope the identity of the end user and/or the subscriber (as both could be different) and making sure that end user experience of using the API is not compromised while doing so.<br><br>**Input params**<br>N/A<br><br>**Output params**<br>N/A<br><br>**Notifications** <br> N/A <br><br>**Notes**<br>APIs around managing consent are expected to be contributed in this subproject. Certain concepts from this subproject will be contributed to the [Commonalities WG](https://github.com/camaraproject/WorkingGroups/tree/main/Commonalities). |
-| Technical viability | N/A | 
+| Technical viability | Based on Mobile Connect Identity APIs | 
```

Comment:
The aim of this new Sub-Project is not only to continue the very good discussions held in https://github.com/camaraproject/WorkingGroups/issues/60 (PR https://github.com/camaraproject/WorkingGroups/pull/68), but also to decide whether new APIs are needed or not for consent management. There does not exist a clear standpoint on this needed yet -- that's precisely one of the outcomes we want to get from the work in this Sub-Project. Therefore, I suggest having this discussion if the proposal gets finally approved